tools/README.md

```diff
@@ -362,4 +362,21 @@ For example, in order to translate a windows-related Sigma rule, one would use:
 
 ```bash
 tools/sigmac -t devo -c tools/config/devo-windows.yml rules/windows/sysmon/sysmon_wmi_susp_scripting.yml
-```
\ No newline at end of file
+```
+
+### Datadog
+The Datadog backend currently supports converting Sigma files to the [Log Search Syntax](https://docs.datadoghq.com/logs/explorer/search_syntax/)
+with the identifier `datadog-logs`. This query can be used in the Security Monitoring product.
```

Comment:
> This query can be used in the Security Monitoring product.

Tidbit: The index facet cannot be used in Security Monitoring. I'm unsure of the technical reason - but the UI will give a helpful message stating that fact. This may cause some confusion for users.

Reply:
Thanks! We added a note here: 309a5629aec4326d8ee24d7adbc1b2d80ace922a